bindings/java/jni/auto_db_java.cpp

```diff
@@ -1237,7 +1233,7 @@ TskAutoDbJava::addUnallocatedPoolBlocksToDb(size_t & numPool) {
         if (m_poolOffsetToVsId.find(pool_info->img_offset) == m_poolOffsetToVsId.end()) {
             tsk_error_reset();
             tsk_error_set_errno(TSK_ERR_AUTO_DB);
-            tsk_error_set_errstr("Error addUnallocatedPoolBlocksToDb() - could not find volume system object ID for pool at offset %lld", (long long int)pool_info->img_offset);
+            tsk_error_set_errstr("Error addUnallocatedPoolBlocksToDb() - could not find volume system object ID for pool at offset %jd", (intptr_t)pool_info->img_offset);
```

Comment:
does TSK have something like `PRIdOFF` for `%jd` since it is not (always) portable.

Reply:
The compiler did not complain about it. I will test the execution as soon as I can.

Reply:
> the compiler did not complain about it. 

it likely will surface only on Windows